lib/wsdl.js

```diff
@@ -1130,7 +1130,7 @@ WSDL.prototype._processNextInclude = function(includes, callback) {
     includePath = url.resolve(self.uri, include.location);
   }
 
-  open_wsdl(includePath, this.options, function(err, wsdl) {
```

Comment:
Should we add the same code to open_wsdl, reasoning for moving it out was to limit it to internal invocation